ui/typings/JQuery.ts

```diff
@@ -0,0 +1,68 @@
+interface JQuery {
```

Comment:
Does the directory **need** to be called `typings`? If not can we rename it to `typing` ?

Reply:
I really thought I would have more of them in the folder. I will rename it :)